app/views/layouts/_noscript_notice.html.erb

```diff
@@ -1,7 +1,7 @@
 <noscript>
   <div class="navbar" style="padding: 12px; margin-top: -20px; background: #ff2211; border-bottom: 1px solid #ffbbaa; text-align: center; text: black;">
     <div class="container">
-      Please enable Javascript for the site to function correctly. We promise not to do anything bad!
+      <%= t '.paragraph_1' %>
```

Comment:
Yeah, I can change it. it's obviously ambiguous in the erb file, but in the
localization file it's named en.views.layouts.noscript_notice.paragraph_1

On Tue, Apr 28, 2015, 10:59 PM Andrew Brown <notifications@github.com>
wrote:

> In app/views/layouts/_noscript_notice.html.erb
> <https://github.com/drusepth/Indent/pull/433#discussion_r29308948>:
>
> > @@ -1,7 +1,7 @@
> >  <noscript>
> >    <div class="navbar" style="padding: 12px; margin-top: -20px; background: #ff2211; border-bottom: 1px solid #ffbbaa; text-align: center; text: black;">
> >      <div class="container">
> > -      Please enable Javascript for the site to function correctly. We promise not to do anything bad!
> > +      <%= t '.paragraph_1' %>
>
> Could probably use a more descriptive name, but this'll do just fine. :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drusepth/Indent/pull/433/files#r29308948>.
>